extensions/wikia/AffiliateService/js/units.js

```diff
@@ -1,5 +1,16 @@
 define('ext.wikia.AffiliateService.units', [], function () {
 	var units = [
+		{
+			campaign: 'hbomax',
+			category: 'hbomax',
+			name: 'hbomax-unit',
+			country: ['US'],
+			heading: 'Where HBO Meets So Much More',
+			subheading: 'START YOUR FREE TRAIL',
+			link: 'https://www.hbomax.com/gzm',
+			image: 'https://static.wikia.nocookie.net/05a5bbb9-7662-416e-99c7-6e6700af9aa3',
+			extra_disclaimer: "Free trail for new customers only. Restrictions Apply."
```

Comment:
Looks like a typo: `trail => trial`

Reply:
No, it's Trail... as in something that you walk down to get to another place...

;-)